core_api/src/lib.rs

```diff
@@ -203,6 +225,38 @@ mod tests {
         };
     }
 
```

Comment:
You should be able to uncomment:
                // TODO #61 - Should instantiation also initialize?
                // assert!(hc.instance.state().nucleus().has_initialized());

Reply:
done